jobs/release-monitoring/discovery/github/Jenkinsfile

```diff
@@ -209,13 +209,17 @@ def installationGitUrl = params.installationGitUrl ?: 'git@github.com:integr8ly/
 def installationGitRef = params.installationGitRef ?: 'master'
 def githubToken = params.githubToken ?: 'jenkins-github-api-token'
 def githubCredentialsID = params.credentialId ?: 'jenkinsgithub'
-def manifestVar = params.manifestVar
+def releaseTagVar = params.releaseTagVar
+def productVersionVar = params.productVersionVar
```

Comment:
Any way we can make this backward compatible so we can avoid breaking all the other jobs while we update them? Maybe we can set releaseTagVar to manifestVar if the param exists?

```
def releaseTagVar = params.manifestVar ?: params.releaseTagVar
```